pkg/config/config_test.go

```diff
@@ -76,9 +76,9 @@ func TestConfigHelper_ValidateRampConfig(t *testing.T) {
 		{"2", 1, 5, 1, false},
 		{"3", 1, 5, 0, false},
 		{"4", 0, 5, 2, false},
-		{"4", 1, 0, 2, false},
-		{"4", 5, 5, 2, false},
-		{"4", 5, 2, 2, false},
+		{"5", 1, 0, 2, false},
+		{"6", 5, 5, 2, false},
+		{"7", 5, 2, 2, false},
```

Comment:
Is the last one saying min 5 and max of 2? Similarly with "5" a min of 1 and max of 0. Maybe thats fine in the use case but from a "logic" perspective your min > max it seems

Reply:
OK, so "5" is testing for a 0 value in `max` and the last one is testing for `min` > `max`.

Saying that, I can actually improve the code and remove some validations and it would still work.

Reply:
And "2" and "3" are testing the same.